linux/system/cpu_frequency.sls

```diff
@@ -0,0 +1,10 @@
+{%- from "linux/map.jinja" import system with context %}
+{%- if system.cpu_frequency is defined %}
+
+ondemand_service_disable:
```

Comment:
Common way of setting cpufreq govenor is using cpufrequtils, eg.
https://askubuntu.com/questions/523640/how-i-can-disable-cpu-frequency-scaling-and-set-the-system-to-performance